### ISSUE TITLE
→ Accessibility: Components | Add error icon to textFields helper text when has error

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#1794]
## Description

Using only color to distinguish the error in the textFields is a failure of: https://www.w3.org/WAI/WCAG22/Understanding/use-of-color.html

Vivo has already explored the usage of an icon in an extended component:

![image](https://github.com/Telefonica/mistica-design/assets/44420072/121b65d8-e1a8-486f-8421-69f98dbfdf5c)

This definition should be extended to all the textFields in Mística that can display an error.

https://jira.tid.es/browse/OBVIVO-2634

## Tasks

- [x] [Update specs](https://www.figma.com/design/e8d3PkXitUJ1Jwgc9Y3mNc/branch/kCv0rWdDj3wqtRm18vuSjN/%F0%9F%94%B8-Input-Fields-%26-Forms-Component-Specs?m=auto&t=FxyyDa7FeAwat3LO-1)
- [ ] Design documentation
- [x] [Add to mobile library](https://www.figma.com/design/WCkDDzlXE16R6yXaljxddj/branch/tu27D9vZCUZON3hfnoMifl/M%C3%ADstica-Mobile?m=auto&t=bSXTr5L5qtZn37ri-1)
- [x] [Add to desktop library](https://www.figma.com/design/DSWhPLyJzbliP1fBrLxDUR/branch/QJoCauhs3HjIFYU271vWz0/M%C3%ADstica-Desktop?m=auto&t=Gs0hjomGyMcoc9ok-1)
- [ ] Add to UI Kit
- [x] [Android ticket](https://jira.tid.es/browse/ANDROID-15004)
- [x] [iOS ticket](https://jira.tid.es/browse/IOS-10449)
- [x] [Web ticket](https://jira.tid.es/browse/WEB-1958)
